pagopa/io-app
# feat: [IOBP-375] New wallet onboarding error outcome handling

## ⚠️ This PR depends on #5155 and https://github.com/pagopa/io-dev-api-server/pull/314

## Short description
This PR adds the failure outcome handling showing the right result screen for each outcome.

## List of changes proposed in this pull request
- Updated wallet definitions;
- Added locales keys;
- Removed previous and deprecated `GenericErrorComponent`;
- Adapted the `WalletOnboardingError` component with `OperationResultScreenContent` that shows the correct pictogram, title and button(s) based on the outcome;

## How to test
- Checkout this PR https://github.com/pagopa/io-dev-api-server/pull/314 from io-dev-api-server
- Start the new wallet onboarding from profile tab and as soon as the webview is showed, select from the dropdown list an error outcome and check if every outcome has the right page following **[this design](https://www.figma.com/file/FSSrvrUrkAaCxsvse1oFFH/Onboarding-metodi?type=design&node-id=1626-2445&mode=design&t=MaX3gAjzLmLuPf9B-4)**

## Preview

https://github.com/pagopa/io-app/assets/34343582/92224f9d-73d2-40dc-8428-f31bbbd2e702
